src/components/BlogPost.js

```diff
@@ -67,39 +77,45 @@ class BlogPost extends React.Component {
                     }
                 </div></h5>
                 <h6 className="card-subtitle mb-2 text-muted">
-                    <a href={"/user/view/" + post.owner}>{post.username}</a>
+                    <a href={"/user/view/" + post.owner}>{post.dev=='true' ? 'Dekkaru' : post.username}</a>
```

Comment:
We really need to implement user roles instead of throwing Dekkaru all around the code lol

Reply:
This is a bit different though, as this just codes in the case of what to display for names for devblog posts

Reply:
For now that's fair, I think such a system is outside the scope of this PR anyway